mockup-service-report/src/main/java/com/itsol/mockup/repository/impl/EmployeeRepositoryImpl.java

```diff
@@ -28,11 +28,12 @@
  */
 
 @Repository
-public class UsersRepositoryImpl implements UsersRepositoryCustom {
-    private static final Logger logger = LogManager.getLogger(UsersRepositoryImpl.class);
+public class EmployeeRepositoryImpl implements EmployeeRepositoryCustom {
```

Comment:
line 55->59, 64->68 should be written under a method

Reply:
ok